cli/src/cli_util.rs

```diff
@@ -50,6 +49,8 @@ use indexmap::IndexMap;
 use indexmap::IndexSet;
 use indoc::writedoc;
 use itertools::Itertools as _;
+use jiff::tz::TimeZone;
+use jiff::Timestamp;
```

Comment:
nit: use qualified `jiff::Timestamp` to avoid future name conflicts with our `Timestamp` type?